torch_geometric/profile/profile.py

```diff
@@ -195,3 +197,37 @@ def rename_profile_file(*args):
         timeline_file += '-' + arg
     timeline_file += '.json'
     os.rename('timeline.json', timeline_file)
+
+
+def e2e_time():
```

Comment:
No need for the double nested function here.

Reply:
How is this different from the `timeit` function?

Reply:
`timeit` is only for CUDA, I don't know how to use the event in CPU. And lightaime suggested to write a new one. Now, `e2e_time` is not necessary.

Reply:
We can probably adjust `timeit` for both CPU and GPU?

Reply:
Yes. The function of `timeit` and `e2e_time` is similar. I'm not quite sure about the difference between `torch.cuda.Event` and  `time.time()` both with `torch.cuda.synchronize()`. If they are the same, the only difference is return values.

Reply:
I think you can use `torch.cuda.Event` in the CPU case too ... but not if there is no CUDA device.  So we could fallback on https://docs.python.org/3/library/timeit.html when there is no CUDA device?

Reply:
Yes. We need to cover the case without CUDA device. So we can use timeit.default_timer to record e2e time, right?